terraspace/.tflint.hcl

```diff
@@ -6,7 +6,7 @@ config {
 
 plugin "aws" {
   enabled = true
-  version = "0.13.2"
+  version = "~> 0.13.3"
```

Comment:
`~>` and other types of constraints [are not supported](https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/plugins.md#version)

Reply:
That sucks.. but done!